components/server/src/container-module.ts

```diff
@@ -299,3 +301,9 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
 
     bind(UsageService).toSelf().inSingletonScope();
 });
+
+function newApplicationClusterProvider(): IApplicationClusterProvider {
+    return {
+        getApplicationCluster: () => process.env.GITPOD_INSTALLATION_SHORTNAME ?? "",
```

Comment:
Can we read from Config here? See [here](https://github.com/gitpod-io/gitpod/pull/14084/files#diff-50198a96e236289d363681353c778e85437ddb54c73d6d2bd61b527a3918bc71R281) for an example.